public/index.html

```diff
@@ -13,38 +13,38 @@
-            <div class="col-sm-4">
-                <a ng-href="#/" class="svg" ng-click="searchCtrl.clear()">
-                    <object type="image/svg+xml" data="logo.svg">Your browser does not support SVG</object>
-                </a>
-            </div>
-            <div class="col-sm-4"></div>
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-sm-12">
+            <a ng-href="#/" class="svg logo" ng-click="searchCtrl.clear()">
+                <object type="image/svg+xml" data="logo.svg">Your browser does not support SVG</object>
+            </a>
         </div>
-        <div class="row">
-            <div class="col-md-offset-3 col-md-6 col-xs-12">
-                <div class="input-group" style="width: 100%">
-                    <form class="form-horizontal" ng-submit="searchCtrl.submit()">
-                        <input type="text" class="form-control" maxlength="50" placeholder="Stock name" ng-model="searchCtrl.query" />
-                        <br>
-                        <br>
-                        <input type="submit" class="btn btn-primary center-block" value="Search" />
-                    </form>
-                </div>
+    </div>
+    <div class="row">
+        <div class="col-md-offset-3 col-md-6 col-xs-12">
+            <div class="input-group" style="width: 100%">
+                <form class="form-horizontal" ng-submit="searchCtrl.submit()">
+                    <input type="text" class="form-control" maxlength="50" placeholder="Stock name"
+                           ng-model="searchCtrl.query"/>
+                    <br>
+                    <br>
+                    <input type="submit" class="btn btn-primary center-block" value="Search"/>
+                </form>
             </div>
         </div>
-        <br>
-        <div ng-view></div>
     </div>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular-route.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular-resource.js"></script>
-    <script src="app.js"></script>
-    <script src="thumbnails-controller.js"></script>
-    <script src="search-controller.js"></script>
-    <script src="stock-controller.js"></script>
-    <script src="quandl-service.js"></script>
+    <br>
+
+    <div ng-view></div>
+</div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular-route.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.8/angular-resource.js"></script>
+<script src="app.js"></script>
+<script src="thumbnails-controller.js"></script>
+<script src="search-controller.js"></script>
+<script src="stock-controller.js"></script>
+<script src="quandl-service.js"></script>
 </body>
 </html>
```

Comment:
because every html has html and body, so you dont want to have all your code always indented by 2 tabs. Anyway, there is possibly some other issue, so hold on with merging either way :)